cluster/kubemark/config-default.sh

```diff
@@ -25,7 +25,11 @@ GCLOUD=gcloud
 ZONE=${KUBE_GCE_ZONE:-us-central1-b}
 REGION=${ZONE%-*}
 NUM_NODES=${NUM_NODES:-100}
+# KUBEMARK_NUM_NODES overrides NUM_NODES
+NUM_NODES=${KUBEMARK_NUM_NODES:-$NUM_NODES}
```

Comment:
I'd probably go with `NUM_NODES=${KUBEMARK_NUM_NODES:-${NUM_NODES:-100}}` and the same with `MASTER_SIZE`.